examples/meta/src/distance/sparseeuclidean.sg

```diff
@@ -0,0 +1,20 @@
+CSVFile f_feats_a("../../data/fm_train_sparsereal.dat")
```

Comment:
wrong file name.... should be `sg.in`

Reply:
shogun also use `SHOGUN_DATA` instead of relative path

Reply:
I don't understand.
This was the filename in the existing python example as well.
Also, I don't see any `sg.in` file in the `data` directory.

Reply:
it should be `examples/meta/src/distance/sparseeuclidean.sg.in` you'll see that all other examples have that extension....

and plz check the other examples for how the data file path is being defined.

Reply:
and as a general rule it'd be great if you could test (compile and run) any patch *before* you create a pull request

Reply:
@Hephaestus12 just add make it more clear, @vigsterkr was not talking about the filename in the csv file, but about the filename of the meta example listing itself